server/routes/overview/overviewView.ts

```diff
@@ -51,6 +52,9 @@ type RenderArgs = {
     problemRetrievingData: boolean
     isPostInduction: boolean
   }
+  reviewStatus: string
+  reviewDateTo: Date | null
+  releaseDate: Date
```

Comment:
I don't think we need `releaseDate` as a separate property here because `prisonerSummary` is already part of the rendered data and that already includes `releaseDate`. I don't think we need to duplicate it do we?

Also, I would put the other 2 fields in an object which contains `problemRetrievingData` (because we might have had a problem retrieving the action plan data and therefore should not attempt to show Review based stuff on the Overview page because we don't know the data). I would also use a enum-like values for the status.
Something like this:
```
    ....
    actionPlanReview: {
      reviewStatus: 'DUE' | 'OVERDUE' | 'NO_SCHEDULED_REVIEW'
      reviewDueDate: Date | null
    }
    ....
```